crates/core/src/host.rs

```diff
@@ -43,7 +43,7 @@ pub struct HostData {
     pub link_definitions: Vec<InterfaceLinkDefinition>,
     /// list of cluster issuers
     pub cluster_issuers: Vec<ClusterIssuerKey>,
-    /// Merged named configuration set for this provider at runtime
+    /// Merged named configuration set for this provider at runtime, indexed by link name
```

Comment:
This still isn't quite right, the `config` is actually just one single HashMap that's materialized from the named configurations given at runtime.

So for example, I'd start a provider by doing something like this:
```bash
wash config put http_reasonable_default PORT=8080
wash start provider wasmcloud.azurecr.io/httpserver:v1 my_httpserver_id --config http_reasonable_default
```

At runtime, the idea is that the HTTP server would startup, and in it's initialization config it would get 
```json
{
    "PORT": "8080"
}
```

And the provider can use that info to set a global default port, for example. Does that make sense?

Reply:
Ah forgot to take this line out -- will remove, this is from back when it was `configs`!

The set up make sense to me, it's the merging them together, and how a single provider can serve multiple clients with multiple configs that's a bit fuzzy for me. But this actual comment is just old

Reply:
So this is fixed now -- I removed the addition here. I can't merge this though until https://github.com/wasmCloud/wasmCloud/pull/1647 lands.